MAUI/Autocomplete/Searching-Filtering.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: Searching and Filtering in .NET MAUI Autocomplete control | Syncfusion
+description: Learn how to perform searching and filtering operations in Syncfusion .NET MAUI Autocomplete control and more.
+platform: maui
+control: SfAutocomplete
+documentation: ug
+---
+
+# Searching and Filtering in .NET MAUI Autocomplete (SfAutocomplete)
+
+The `Autocomplete` control allows for rich text searching and filtering functionality.
+
+## Searching based on member path
+
+The `DisplayMemberPath` and `TextMemberPath` properties of Autocomplete control specify the property path, by which the searching must be done when a custom data is bound to the `ItemsSource` property.
+
+ * `DisplayMemberPath` - Specifies the property path whose value is displayed as text in the drop-down menu. The default value is `string.Empty`.
+
+ * `TextMemberPath` - Specifies the property path whose value is used to perform searching based on user input received in the selection box portion of the `Autocomplete` control. The default value is `string.Empty`. When `TextMemberPath` is `null` or `string.Empty`, searching will be performed based on `DisplayMemberPath`. 
+
+N> `DisplayMemberPath` and `TextMemberPath` will be effective for the collection item that holds two or more properties in it.
+
+N> When both the `DisplayMemberPath` and `TextMemberPath` properties have a `null` or `string.Empty` value, searching will be performed based on the class name with namespace of the item.
```

Comment:
N> When both the `DisplayMemberPath` and `TextMemberPath` properties have a `null` or `string.Empty` value, searching will be performed based on the class name with the namespace of the item.